inference-engine/ie_bridges/c/samples/object_detection_sample_ssd/main.c

```diff
@@ -167,7 +167,7 @@ void parseInputFilesArguments(int argc, char **argv) {
         readInputFilesArgument(argv[i]);
     }
 
-    if (file_num) {
+    if (file_num < 20) {
```

Comment:
What does this number mean? Maybe: int max_files = 20; if (file_num < max_files); ?